openshift-gitops/overlays/nerc-ocp-infra/patches/argocds/openshift-gitops.yaml

```diff
@@ -18,3 +18,12 @@ spec:
       - "ManagedClusterInfo"
       clusters:
       - "*"
+  rbac:
```

Comment:
The linked document says to create a `ConfigMap` called `argocd-rbac-cm` and that would include the policy. How did you figure out we can include `rbac` in this resource of kind `ArgoCD`? I can't seem to find this in the documents.

Reply:
You need to look at the docs for the [ArgoCD operator](https://argocd-operator.readthedocs.io/en/latest/reference/argocd/) or for the [OpenShift GitOps operatpr](https://docs.openshift.com/container-platform/4.10/cicd/gitops/argo-cd-custom-resource-properties.html).